data/json/items/obsolete.json

```diff
@@ -3324,5 +3324,120 @@
+      },
+      {
+        "id": "scar_l",
+        "name": { "str": "FN SCAR-L" },
+        "description": "A highly accurate and modular assault rifle specially designed for the United States Special Operations Command.  The 'L' in its name stands for light, as it uses the lightweight .223 round.",
+        "ascii_picture": "scar-L"
+      },
+      {
+        "id": "m38dmr",
+        "name": { "str": "M38 DMR" },
+        "description": "The M38 Designated Marksman Rifle is an M27 IAR, itself derived from a H&K416, selected for accuracy, and outfitted with a variable power scope and a QDSS suppressor."
+      },
+      {
+        "id": "m4_cqbr",
+        "name": { "str": "MK 18 CQBR" },
+        "description": "This is a shorter M4 carbine with a 10.3 inch barrel, intended for confined spaces and close quarters battle situations."
+      }
+    ],
+    "to_hit": -1,
+    "bashing": 12,
+    "material": [ "steel", "plastic" ],
+    "symbol": "(",
+    "color": "dark_gray",
+    "ammo": [ "223" ],
+    "ranged_damage": { "damage_type": "bullet", "amount": -1 },
+    "dispersion": 160,
+    "durability": 8,
+    "min_cycle_recoil": 1350,
+    "modes": [ [ "DEFAULT", "semi-auto", 1 ], [ "AUTO", "auto", 4 ] ],
+    "barrel_volume": "250 ml",
+    "built_in_mods": [ "folding_stock" ],
+    "pocket_data": [
+      {
+        "pocket_type": "MAGAZINE_WELL",
+        "holster": true,
+        "max_contains_volume": "20 L",
+        "max_contains_weight": "20 kg",
+        "item_restriction": [
+          "stanag30",
+          "stanag5",
+          "stanag10",
+          "stanag20",
+          "stanag40",
+          "stanag50",
+          "stanag60",
+          "stanag60drum",
+          "stanag90",
+          "stanag100",
+          "stanag100drum",
+          "stanag150",
+          "survivor223mag"
+        ]
+      }
+    ]
+  },
+  {
+    "id": "sig552",
+    "copy-from": "nato_assault_rifle",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "//": "This could use a better name, I just wanted to differentiate it from assault rifles, as it also has a burst mode",
+    "name": { "str": "NATO assault rifle with burst", "str_pl": "NATO assault rifles with burst" },
+    "description": "An assault rifle used by a NATO member state.  Unlike other assault rifles, this one has a burst fire mode in addition to semi-automatic and fully-automatic modes.  As per NATO's STANAG agreement, it is chambered in 5.56x45mm.",
+    "variants": [
+      {
+        "id": "sig552",
+        "name": { "str": "SIG 552" },
+        "description": "A compact selective fire automatic rifle designed for the Swiss military.  It features a three-round burst mode and an integrated folding stock.",
+        "weight": 1
+      }
+    ],
+    "built_in_mods": [ "folding_stock" ],
+    "modes": [ [ "DEFAULT", "semi-auto", 1 ], [ "BURST", "3 rd.", 3 ], [ "AUTO", "auto", 4 ] ]
```

Comment:
These shouldn't be obsoleted, but the infrastructure to do what should happen doesn't exist yet. I'll handle that.

Reply:
Oh, alright, was suggested to do so yesterday. Should I just remove those lines then?

Reply:
Keep it as is for now, I'll tell you how to get it sorted when it is possible.

Reply:
Alright